token-service/src/main/java/org/p2p/token/service/database/converter/TokenServiceRoomConverter.kt

```diff
@@ -0,0 +1,12 @@
+package org.p2p.token.service.database.converter
+
+import androidx.room.TypeConverter
+import org.p2p.token.service.model.TokenServiceNetwork
+
+internal object  TokenServiceRoomConverter {
+    @TypeConverter
+    fun tokenServiceNetworkToString(value: TokenServiceNetwork?) = value?.networkName
+
+    @TypeConverter
+    fun stringToTokenServiceNetwork(value: String) = TokenServiceNetwork.valueOf(value)
```

Comment:
tokenServiceNetworkToString (TokenServiceNetwork?) -> String?

stringToTokenServiceNetwork doesn't accept any nullable value which is strange, so I think TypeConverter won't be used